

[WARN ] 2019-11-03 21:37:54,464 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1108)
Client session timed out, have not heard from server in 20000ms for sessionid 0x16e2aebdeea0094


[WARN ] 2019-11-04 01:56:11,189 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e2aebdeea0094 for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-04 02:03:32,322 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:160)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=18464&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1572772540239, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=18464&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1572772540239) failed to connect to server /192.168.25.1:20882, error message is:connection timed out: /192.168.25.1:20882
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.jboss.netty.channel.ConnectTimeoutException: connection timed out: /192.168.25.1:20882
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:137)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-04 02:03:49,797 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e2aebdeea0094 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-04 02:03:54,490 method:org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1285)
Unable to reconnect to ZooKeeper service, session 0x16e2aebdeea0094 has expired


[WARN ] 2019-11-04 02:03:56,873 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,360 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,361 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,362 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,362 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,363 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,364 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,365 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,366 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,492 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,494 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,494 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,495 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,498 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,498 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,645 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,645 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,646 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,647 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:57,968 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 02:03:59,238 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 08:17:17,168 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e2aebdeea00ae for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-04 08:17:30,738 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:160)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=18464&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1572772540239, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=18464&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1572772540239) failed to connect to server /192.168.25.1:20882 client-side timeout 3000ms (elapsed: 12429ms) from netty client 192.168.25.1 using dubbo version 2.5.3
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:127)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 08:17:47,727 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e2aebdeea00ae for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-04 08:17:56,212 method:org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1285)
Unable to reconnect to ZooKeeper service, session 0x16e2aebdeea00ae has expired


[WARN ] 2019-11-04 13:06:56,714 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1108)
Client session timed out, have not heard from server in 20000ms for sessionid 0x16e2aebdeea00ba


[WARN ] 2019-11-04 16:43:06,563 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1108)
Client session timed out, have not heard from server in 20000ms for sessionid 0x16e2aebdeea00ba


[WARN ] 2019-11-04 21:12:06,880 method:com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:221)
 [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.5.3, current host: 192.168.25.1
java.io.IOException: Can not lock the registry cache file C:\Users\Lenovo\.dubbo\dubbo-registry-192.168.25.128.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 22:08:36,975 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e2aebdeea0117 for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-04 22:09:00,621 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:160)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=21140&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1572874148598, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=21140&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1572874148598) failed to connect to server /192.168.25.1:20882 client-side timeout 3000ms (elapsed: 22434ms) from netty client 192.168.25.1 using dubbo version 2.5.3
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:127)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-04 22:09:17,614 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e2aebdeea0117 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-04 22:09:33,795 method:org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1285)
Unable to reconnect to ZooKeeper service, session 0x16e2aebdeea0117 has expired


[WARN ] 2019-11-04 23:23:58,868 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1108)
Client session timed out, have not heard from server in 20001ms for sessionid 0x16e2aebdeea0141


[WARN ] 2019-11-04 23:24:21,537 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e2aebdeea0141 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-04 23:24:44,574 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e2aebdeea0141 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-06 21:54:53,571 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0016 for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 00:13:39,218 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0016 for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[ERROR] 2019-11-07 03:13:40,091 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:155)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:Network is unreachable: no further information
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.SocketException: Network is unreachable: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more


[WARN ] 2019-11-07 03:13:40,954 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0016 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 03:13:42,535 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:160)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:No route to host: no further information
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-07 03:13:42,583 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0016 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 03:14:22,133 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0016 for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[ERROR] 2019-11-07 11:46:17,232 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:155)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882 client-side timeout 3000ms (elapsed: 30714668ms) from netty client 192.168.25.1 using dubbo version 2.5.3
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:127)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-07 11:46:18,721 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0016 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.SocketException: Network is unreachable: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 11:46:19,522 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:160)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:No route to host: no further information
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-07 11:46:20,009 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0016 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 11:46:22,089 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0016 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 11:46:44,463 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0016 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 11:47:01,590 method:org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1285)
Unable to reconnect to ZooKeeper service, session 0x16e40a6a28e0016 has expired


[WARN ] 2019-11-07 13:14:29,963 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0031 for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[ERROR] 2019-11-07 14:05:02,574 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:155)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:connection timed out: /192.168.25.1:20882
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.jboss.netty.channel.ConnectTimeoutException: connection timed out: /192.168.25.1:20882
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:137)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-07 14:05:20,620 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0031 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 14:05:22,613 method:org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1285)
Unable to reconnect to ZooKeeper service, session 0x16e40a6a28e0031 has expired


[WARN ] 2019-11-07 15:01:57,349 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e003f for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[ERROR] 2019-11-07 18:01:58,138 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:155)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:No route to host: no further information
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-07 18:01:59,147 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e003f for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 18:02:00,725 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:160)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:No route to host: no further information
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-07 18:02:00,726 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e003f for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 18:02:02,052 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e003f for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 18:02:33,763 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e003f for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[ERROR] 2019-11-07 22:01:27,826 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:155)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:connection timed out: /192.168.25.1:20882
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.jboss.netty.channel.ConnectTimeoutException: connection timed out: /192.168.25.1:20882
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:137)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-07 22:01:28,335 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e003f for server null, unexpected error, closing socket connection and attempting reconnect
java.net.SocketException: Network is unreachable: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 22:01:29,869 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:160)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:No route to host: no further information
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-07 22:01:30,266 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e003f for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 22:01:53,033 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e003f for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 22:02:15,240 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e003f for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-07 22:02:16,732 method:org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1285)
Unable to reconnect to ZooKeeper service, session 0x16e40a6a28e003f has expired


[WARN ] 2019-11-08 00:24:47,881 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0048 for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[ERROR] 2019-11-08 03:24:48,542 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:155)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:connection timed out: /192.168.25.1:20882
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.jboss.netty.channel.ConnectTimeoutException: connection timed out: /192.168.25.1:20882
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:137)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-08 03:24:50,114 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0048 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-08 03:24:50,548 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:160)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:No route to host: no further information
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-08 03:24:52,055 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0048 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-08 03:25:26,688 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0048 for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-08 15:18:41,275 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0048 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.SocketException: Network is unreachable: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[ERROR] 2019-11-08 15:18:41,276 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:155)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:connection timed out: /192.168.25.1:20882
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.jboss.netty.channel.ConnectTimeoutException: connection timed out: /192.168.25.1:20882
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:137)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-08 15:18:45,595 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0048 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-08 15:18:46,779 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0048 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-08 15:18:48,647 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0048 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-08 15:19:04,926 method:org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1285)
Unable to reconnect to ZooKeeper service, session 0x16e40a6a28e0048 has expired


[WARN ] 2019-11-08 16:06:24,932 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0057 for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[ERROR] 2019-11-08 17:00:18,583 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:155)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:connection timed out: /192.168.25.1:20882
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.jboss.netty.channel.ConnectTimeoutException: connection timed out: /192.168.25.1:20882
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:137)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-08 17:00:20,026 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0057 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-08 17:00:36,352 method:org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1285)
Unable to reconnect to ZooKeeper service, session 0x16e40a6a28e0057 has expired


[WARN ] 2019-11-08 19:32:17,421 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0061 for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-08 19:50:54,237 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0061 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-08 19:50:59,141 method:org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1285)
Unable to reconnect to ZooKeeper service, session 0x16e40a6a28e0061 has expired


[WARN ] 2019-11-08 22:33:28,565 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e006f for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[ERROR] 2019-11-09 01:33:27,953 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:155)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:No route to host: no further information
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-09 01:33:29,257 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e006f for server null, unexpected error, closing socket connection and attempting reconnect
java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-09 01:33:29,961 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:160)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:No route to host: no further information
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-09 01:33:52,219 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e006f for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-09 01:33:54,688 method:org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1285)
Unable to reconnect to ZooKeeper service, session 0x16e40a6a28e006f has expired


[WARN ] 2019-11-09 01:33:58,352 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0076 for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[ERROR] 2019-11-09 09:38:03,697 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:155)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:connection timed out: /192.168.25.1:20882
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.jboss.netty.channel.ConnectTimeoutException: connection timed out: /192.168.25.1:20882
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:137)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-09 09:38:04,954 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1108)
Client session timed out, have not heard from server in 29044945ms for sessionid 0x16e40a6a28e0076


[WARN ] 2019-11-09 09:38:06,367 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:160)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882, error message is:No route to host: no further information
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more


[WARN ] 2019-11-09 09:38:23,364 method:org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1285)
Unable to reconnect to ZooKeeper service, session 0x16e40a6a28e0076 has expired


[WARN ] 2019-11-09 11:15:18,261 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0087 for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-09 11:15:26,595 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:160)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882 client-side timeout 3000ms (elapsed: 7775ms) from netty client 192.168.25.1 using dubbo version 2.5.3
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:127)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-09 11:15:43,596 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0087 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-09 11:15:52,241 method:org.apache.zookeeper.ClientCnxn$SendThread.onConnected(ClientCnxn.java:1285)
Unable to reconnect to ZooKeeper service, session 0x16e40a6a28e0087 has expired


[WARN ] 2019-11-09 11:16:01,632 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0091 for server 192.168.25.128/192.168.25.128:2181, unexpected error, closing socket connection and attempting reconnect
java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:384)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-09 11:18:26,261 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:160)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882 client-side timeout 3000ms (elapsed: 143919ms) from netty client 192.168.25.1 using dubbo version 2.5.3
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:127)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


[WARN ] 2019-11-09 11:18:41,550 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0091 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-09 11:18:43,476 method:org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1162)
Session 0x16e40a6a28e0091 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.SocketException: Network is unreachable: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:744)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)


[WARN ] 2019-11-09 11:19:57,590 method:com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:160)
 [DUBBO] client reconnect to 192.168.25.1:20882 find error . url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006, dubbo version: 2.5.3, current host: 192.168.25.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.25.1:20882/com.pekka.search.service.SearchService?anyhost=true&application=pekka-search-web&check=false&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.pekka.search.service.SearchService&methods=search&pid=5544&revision=0.0.1-SNAPSHOT&side=consumer&timeout=300000&timestamp=1573045739006) failed to connect to server /192.168.25.1:20882 client-side timeout 3000ms (elapsed: 73319ms) from netty client 192.168.25.1 using dubbo version 2.5.3
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:127)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
